source/features/align-issue-labels.tsx

```diff
@@ -12,5 +14,8 @@ void features.add({
 }, {
 	waitForDomReady: false,
 	repeatOnAjax: false,
```

Comment:
any reason this is not working correctly with `repeatOnAjax: false`

Reply:
Because sometimes it goes from `isPR` to `isConversationList` with an ajax load, but `repeatOnAjax: false` prevents the page detection